code/database/behavior/orderable.php

```diff
@@ -192,7 +189,7 @@ protected function _beforeInsert(DatabaseContextInterface $context)
      */
     protected function _beforeUpdate(DatabaseContextInterface $context)
     {
-        if(isset($this->order) && $this->hasProperty('ordering')) {
+        if(isset($this->order)) {
```

Comment:
The refactor doesn't touch that, it only removes the additional check for 'ordering'.